pkg/runtime/resolver.go

```diff
@@ -59,25 +59,23 @@ type resolver struct {
 	// rules in the configuration database keyed by $namespace.
 	rules map[string][]*Rule
 
-	// Opaque ruleState used in this configuration
-	// After the resolver has no references, this state can be
-	// cleaned up correctly by the caller.
-	ruleState interface{}
-
 	// refCount tracks the number requests currently using this
 	// configuration. resolver state can be cleaned up when this count is 0.
 	refCount int32
+
+	// ID of the resolver for debugging.
+	ID int
```

Comment:
Resolver ID defines the monotonically increasing ID for the config snapshot. 
It is used in logging now. I think it is useful to know it in the logs.